CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 
 ## 0.15.0-dev
 
+### Added
+
+- AlwaysOnTop window level provides always on top window placement on platforms that support it.
```

Comment:
This should document the new config option, like `window.level` to set preferred window level.